packages/api/src/index.js

```diff
@@ -31,8 +31,8 @@ router.post('/car',                 auth['🔒'](carPost))
 router.put('/car/:cid',             auth['🔒'](carPut))
 router.post('/upload',              auth['🔒'](uploadPost))
 router.get('/user/uploads',         auth['🔒'](userUploadsGet))
+router.delete('/user/uploads/:cid', auth['🔒'](userUploadsDelete))
```

Comment:
This is a blocker here. Until we get scoped api tokens as per #314 we deliberately made it so tokens can only be used for non-destructive actions like uploading and listing.